platform/testing/end-to-end/cypress-migration-guide.md

```diff
@@ -254,10 +207,13 @@ To add aXe checks to your tests use the custom `cy.axeCheck()` command based off
 As documented by the plugin, be sure to call `cy.injectAxe()` after each `cy.visit()`, after which `cy.axeCheck()` can be invoked any number of times.
 
 Nightwatch:
+
 ```javascript
-client.axeCheck('.main');
+client.axeCheck(".main");
 ```
+
 Cypress:
+
 ```javascript
 cy.visit(PAGE_URL);
 cy.injectAxe();
```

Comment:
Does it make sense to use Holden's custom helper that consolidates the `injectAxe()` and `axeCheck()` steps?

Reply:
It definitely does! Thanks for pointing this out.